### ISSUE TITLE
Adding a direct polygon mask export

### ISSUE THREAD

**Issue by saskra** [heartexlabs/label-studio-converter#190]
I am working on the same images in the same project with both polygon and more detailed brush masks, which I would like to export and import into Python in the easiest way possible. Right now I'm working on extracting the information I need from the COCO export. However, a welcome simplification for me would be to be able to export both types of labels together with one click as PNG images or even better as Numpy arrays. Would that be possible?

**Comment by makseq** [comment]
It's a little bit unclear: you want to have files with brush masks as numpy arrays and coco export file with polygon coordinates?

**Comment by saskra** [comment]
No, that would be exactly what is already feasible now. I want both as Numpy or PNG export, but unfortunately that doesn't work for polygons at the moment.

**Comment by makseq** [comment]
So, you mean you want to convert polygon coordinates to png/numpy images?

**Comment by saskra** [comment]
Yes, that would be great. In fact, I'm writing myself a function that makes such images and arrays out of the polygons after the COCO export. But I was wondering why this is already offered directly in the GUI for brush labels, but not for polygons.

**Comment by makseq** [comment]
it's because brush labels are pixel-type labeling, it's stored as RLE image and then decoded to png/numpy. Polygons are just arrays of (x,y) coordinates. However, if you could contribute with your converter from polygons to images/numpy, it would be great :-)
